src/app/views/query-runner/request/Request.tsx

```diff
@@ -80,13 +86,35 @@ export class Request extends Component<IRequestComponent, any> {
     );
   }
 
+  private trackTabClickEvent(tabKey: string) {
+    switch (tabKey) {
+      case 'permissions': {
+        telemetry.trackEvent(TAB_CLICK_EVENT, { ComponentName: 'Permissions Tab'});
+        break;
+      }
+      case 'auth': {
+        telemetry.trackEvent(TAB_CLICK_EVENT, { ComponentName: 'Auth Tab'});
+        break;
+      }
+      default: {
+        break;
+      }
+    }
```

Comment:
[nit pick] This can be summarised to a one-liner like
```suggestion
        telemetry.trackEvent(TAB_CLICK_EVENT, { ComponentName: `${tabKey} tab`});
```